src/components/Filter/FilterModal.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC } from 'react';
+import styles from './filter.module.css';
+
+const FilterModal: FC = () => {
+  return (
+    <div className={styles.modalContainer}>
+      <div className={styles.insideModalContainer}>
+        <div className="options">
+          <input
+            type="checkbox"
+            name="openPr"
+            id="filter"
+            className="filterClass"
+            style={{ marginRight: '1rem' }}
+          />
+          <label htmlFor="filter">All</label>
+        </div>
+        <div className="options">
+          <input
+            type="checkbox"
+            name="openPr"
+            id="filter"
+            className="filterClass"
+            style={{ marginRight: '1rem' }}
+          />
+          <label htmlFor="filter">Open</label>
+        </div>
+        <div className="options">
+          <input
+            type="checkbox"
+            name="openPr"
+            id="filter"
+            className="filterClass"
+            style={{ marginRight: '1rem' }}
```

Comment:
inline css why?

Reply:
this is temporary for now, will remove it when will implement logic for filter